### PR TITLE
String added for "OK" Button + Translation [Spanish]

### PR DESCRIPTION
**Pull request by xPoppyx**
- String added for "OK" Button and Translated into spanish
- Strings added with #838 have been translated into Spanish

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@xPoppyx It looks like there are some merge conflicts, can you pull in the latest changes from `master`?

**Comment by xPoppyx** [comment]
I'll fix a couple more things

**Comment by yaichenbaum** [comment]
azp /run

**Comment by xPoppyx** [comment]
I want to add in this PR the translation for the strings added with #838

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -597,6 +597,8 @@
   <data name="StatusBarControlTilesView.Text" xml:space="preserve">
     <value>Tiles View</value>
   </data>
+  <data name="PropertiesDialogOKButton.Content" xml:space="preserve">
+    <value>OK</value>
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:601]: It looks like you are missing a closing tag here.

**Comment by xPoppyx** [comment]
I think it's ok now, and sorry for the mistakes, they happened by not solving the merge conflict correctly

**Comment by yaichenbaum** [comment]
:shipit:

**Pull request merged**